### PR TITLE
Added Dockerfile and k8s config files

### PR DESCRIPTION
**Pull request by base10**

### CODE REVIEW

File: deploy/k8s-app-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      app: "link-app"
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        app: "link-app"
+    spec:
+      containers:
+      - name: "link-app"
+        image: path.to.docker/image/link-app-docker:latest
```
Comment (base10) [inline deploy/k8s-app-deployment.yaml:20]: I take it this is piece still needs to be worked out?

**Review by base10 — COMMENTED** [review]
Overall, looks good. Once I get the `initial-user-model` branch merged in, we'll want to add a Sidekiq file as well. I think we'll also want an example cronjob file.

File: deploy/k8s-app-deployment.yaml (hunk 1 above)
Comment (therogue) [reply deploy/k8s-app-deployment.yaml:20]: Yes, that's right.

**Comment by therogue** [comment]
Merging initial sample k8s files.

**Pull request merged**